ext/travis/test

```diff
@@ -0,0 +1,21 @@
+#!/bin/bash
+
+set -euxo pipefail
+
+if [ "$TRAVIS_OS_NAME" = linux ]; then
+    uid="$(id -u)"
+    gid="$(id -g)"
+    tinyfs="$(pwd)/tinyfs"
+
+    sudo -i dd if=/dev/zero of=tinyfs.img bs=10MB count=1
+    sudo -i chmod 600 tinyfs.img
+    sudo -i mke2fs -F tinyfs.img
+    sudo -i mkdir "$tinyfs"
+    sudo -i mount -oloop tinyfs.img "$tinyfs"
+    sudo -i chown "$uid:$gid" "$tinyfs"
```

Comment:
Ah, I believe you don't get `root` access (at least by default?) on Jenkins; jobs run as the `jenkins` user.

@waynr Would have a better answer